### ISSUE TITLE
Consider a Frictionless Data Guide for Darwin Core

### ISSUE THREAD

**Issue by tucotuco** [tdwg/dwc#448]
Just an idea awaiting demand at this point.

**Comment by tucotuco** [comment]
Yes, the idea would be to produce the Guide in parallel with the guides for
other serializations.

**Comment by andrejjh** [comment]
Thanks Peter for referring to FrictionlessDarwinCore. If I can help on that guide, don't hesitate.

**Comment by tucotuco** [comment]
[The Darwin Core Data Package guide](https://gbif.github.io/dwc-dp/dp/) has been written and presented for public review.

**Comment by yuummmer** [comment]
_For context I’m building a local-first dataset preflight/validator (FAIRy), so I’m very sympathetic to the “what structure do I need?” problem for first-time publishers._

+1 to @Agnie-R  's idea. The Data Mapping Guide already feels close to a wizard since it’s essentially a decision tree: if you have this class of information, which tables (and joins) do you need?

One suggestion that could make this scalable for tooling (including IPT) without reducing DwC-DP’s flexibility is to publish the wizard logic as a machine-readable checklist/decision spec (even a simple YAML/JSON alongside the guide), where each section declares:

- the trigger question (e.g., Do you have stable person/organization entities worth reusing?)
- the tables implied (e.g., Agent and related join tables)
- whether it’s minimum recommended vs optional

Then the docs could render it as a friendly flowchart/table, and software could implement it as a form that outputs suggested tables and joins for a given dataset.

**Comment by tucotuco** [comment]
As the DwC-DP Guide has been written and presented in public review, this issue no longer has anything that needs to be reviewed. Closing.
